docs_nnx/guides/filters_guide.md

```diff
@@ -8,12 +8,18 @@ jupytext:
     jupytext_version: 1.13.8
 ---
 
-# Using Filters
+# Using `Filter`s
```

Comment:
Can remove codefont Markdown formatting from section titles similar to Flax Basics @cgarciae